packages/opentelemetry-instrumentation-grpc/README.md

```diff
@@ -0,0 +1,74 @@
+# OpenTelemetry gRPC Instrumentation for Node.js
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`grpc`](https://grpc.github.io/grpc/node/). Currently, version [`1.x`](https://www.npmjs.com/package/grpc?activeTab=versions) of the Node.js gRPC library is supported.
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node) package.
+
+## Installation
+
+```sh
+npm install --save @opentelemetry/instrumentation-grpc
+```
+
+## Usage
+
+OpenTelemetry gRPC Instrumentation allows the user to automatically collect trace data and export them to the backend of choice, to give observability to distributed systems when working with [gRPC](https://www.npmjs.com/package/grpc).
+
+To load a specific instrumentation (**gRPC** in this case), specify it in the Node Tracer's configuration.
+
+```javascript
+const { NodeTracerProvider } = require('@opentelemetry/node');
+
+const provider = new NodeTracerProvider({
+  // be sure to disable old plugin
+  plugins: {
+    grpc: { enabled: false, path: '@opentelemetry/plugin-groc' }
+  },
+});
+
+const httpInstrumentation = new HttpInstrumentation({
```

Comment:
I think it should be grpc, not http

Reply:
Bad pasting, i will update that tomorrow thanks

Reply:
I've rebased + fixed this, PTAL